.github/workflows/coverage_base.yml

```diff
@@ -0,0 +1,51 @@
+# This is a basic workflow to help you get started with Actions
+
+name: TestCoverage
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the dev branch
+  push:
+    branches: [dev]
```

Comment:
Files are the other way around in this.
`base` should have `master` and `development` should have `dev`.

Reply:
fixed here: 2a72aab